### PR TITLE
plugins:foreman_ansible - Release 0.3

### PR DESCRIPTION
**Pull request by dLobatog**
For 1.9, 1.10 and nightlies.

@domcleal Why does this package now have two 'Provides'?

### CODE REVIEW

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -64,6 +66,7 @@ cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
 %{gem_spec}
 %{foreman_bundlerd_dir}/%{gem_name}.rb
 %exclude %{gem_instdir}/test
+%exclude %{gem_instdir}/extras
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: This dir isn't in the gem, it shouldn't be listed (breaks EL6).

**Comment by domcleal** [comment]
> @domcleal Why does this package now have two 'Provides'?

It always should have.  The rubygem(%name) one is a Fedora standard, to be preferred over rubygem-name when using dependencies.  The foreman-plugin-%name one is something we have on every plugin package as an easier way to install plugins, but we haven't really used it.

**Comment by dLobatog** [comment]
Updated to not exclude the 'extras' directory which is not in the gem.

**Comment by domcleal** [comment]
Thanks @dLobatog, merged and building for nightly, 1.10 and 1.9.

**Pull request merged**